src/components/NcListItem/NcListItem.vue

```diff
@@ -1012,6 +1013,7 @@ export default {
 
 	&-details {
 		&__details {
+			font-size: var(--font-size-small);
 			color: var(--color-text-maxcontrast);
 			margin: 0 9px !important;
 			font-weight: normal;
```

Comment:
Can we group the font related rules?
```suggestion
			color: var(--color-text-maxcontrast);
			font-size: var(--font-size-small);
			font-weight: normal;
			margin: 0 9px !important;
```